### PR TITLE
Implementation of AWS::Kinesis::StreamConsumer for CloudFormation

### PR DESCRIPTION
**Pull request by Morijarti**
## Changes
Implementation of CloudFormation support for AWS::Kinesis::StreamConsumer

<!-- The following sections are optional, but can be useful! 

## TODO

What's left to do:

- Add code for case where Consumer is already registed

### CODE REVIEW

File: localstack/services/kinesis/resource_providers/aws_kinesis_streamconsumer.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        """
+        Fetch resource information
+
+
+        """
+        model = request.desired_state
+        kinesis = request.aws_client_factory.kinesis
+        kinesis.deregister_stream_consumer(ConsumerARN=model["ConsumerARN"])
+
+        model["ConsumerStatus"] = "DELETING"
```
Comment (pinzon) [inline localstack/services/kinesis/resource_providers/aws_kinesis_streamconsumer.py:107]: I'm not sure if this line is necessary.cc/ @dominikschubert @simonrw

File: tests/aws/services/cloudformation/resources/test_kinesis.py [hunk 2]
```diff
@@ -174,3 +174,17 @@ def test_dynamodb_stream_response_with_cf(deploy_cfn_template, aws_client):
     assert response.get("TableName") == "EventTable"
     assert len(response.get("KinesisDataStreamDestinations")) == 1
     assert "StreamArn" in response.get("KinesisDataStreamDestinations")[0]
+
+
+@markers.aws.unknown
```
Comment (pinzon) [inline tests/aws/services/cloudformation/resources/test_kinesis.py:179]: New tests should be AWS validated

File: localstack/services/kinesis/resource_providers/aws_kinesis_streamconsumer.py (hunk 1 above)
Comment (Morijarti) [reply localstack/services/kinesis/resource_providers/aws_kinesis_streamconsumer.py:107]: I don't think it's needed. I'll remove the line.

File: tests/aws/services/cloudformation/resources/test_kinesis.py (hunk 2 above)
Comment (Morijarti) [reply tests/aws/services/cloudformation/resources/test_kinesis.py:179]: On it.

Comment (Morijarti) [reply tests/aws/services/cloudformation/resources/test_kinesis.py]: I've tested against AWS with @simonrw 's help

**Review by pinzon — APPROVED** [review]
Thanks for the changes!

**Pull request merged**